console/src/views/Home.js

```diff
@@ -10,10 +10,12 @@ import Rearrangeable from '@enact/agate/Rearrangeable';
 import CompactRadio from '../components/CompactRadio';
 import CompactHvac from '../components/CompactHVAC';
 import CompactAppList from '../components/CompactAppList';
+import WeatherItem from '../components/WeatherItem';
+import AppStateConnect from '../App/AppContextConnect';
 
 import css from './Home.less';
 
-const allSlotNames = ['bottomLeft', 'bottomRight', 'topLeft', 'topRight'];
+const allSlotNames = ['bottomLeft', 'bottomRight', 'topLeft', 'topRight', 'topMiddle'];
```

Comment:
Please rename `topMiddle` to `topCenter` and place it between `topLeft` and `topRight`.

Reasoning: "middle" in web terminology refers to vertical centering while "center" refers to horizontal centering. Ex: `text-align: center`, `vertical-align: middle;`